### PR TITLE
Adding danger and Dangerfile

### LINKED ISSUE #127: Update travis CI
Integrate awesome_bot with Danger and have results reported back to the pull request.

For reference 
1. [awesome_bot danger documentation](https://github.com/dkhamsing/awesome_bot#danger)
2. [Sample travis CI](https://github.com/vsouza/awesome-ios/blob/master/.travis.yml)

### PR DESCRIPTION
**Pull request by yamilurbina**
Resolves #127

### CODE REVIEW

**Comment by yamilurbina** [comment]
@Rishabh04-02 danger is added but it needs the `DANGER_GITHUB_API_TOKEN` environment variable added to Travis so it can comment on the pull-request. I read it here: http://danger.systems/guides/getting_started.html#setting-up-danger-to-run-on-your-ci

File: Dangerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Check links
+require 'json'
+results = File.read 'ab-results-README.md-markdown-table.json'
+j = JSON.parse results
+if j['error']==true
```
Comment (sotayamashita) [inline Dangerfile]: [Just confirm] need `true`?

if `j['error']`'s is boolean type, you do not need to check with `true`

File: Dangerfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Check links
+require 'json'
+results = File.read 'ab-results-README.md-markdown-table.json'
+j = JSON.parse results
```
Comment (sotayamashita) [inline Dangerfile]: Please do not omit variable name. `j` -> `json` or something else.

**Comment by sotayamashita** [comment]
@yamilurbina I have added the variable in TravisCI

**Comment by Rishabh04-02** [comment]
@yamilurbina are you still working on this?

**Pull request closed without merging**